### ISSUE TITLE
Cannot clone MT32Emu/munt due to updated GitHub security

### ISSUE THREAD

**Issue by shinra-electric** [MaddTheSane/Boxer#41]
When updating the submodules, cloning munt will fail: 

`Submodule 'munt' (git://github.com/munt/munt.git) registered for path 'Vendor/MT32Emu/munt'`
`Cloning into '/Users/xianmo/Downloads/Boxer/Vendor/MT32Emu/munt'...`
`fatal: remote error: `
  `The unauthenticated git protocol on port 9418 is no longer supported.`
`Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.`
`fatal: clone of 'git://github.com/munt/munt.git' into submodule path '/Users/xianmo/Downloads/Boxer/Vendor/MT32Emu/munt' failed`

The reason for the failure is updated security on GitHub which is described [here](https://github.blog/2021-09-01-improving-git-protocol-security-github/#no-more-unauthenticated-git).

I searched a bit for fixes, and I found [this example](https://github.com/che-incubator/che-workspace-telemetry-client/pull/76/files) from another project where they change the git path to github. 

So if `git://github.com/munt/munt.git` is changed to `github:munt/munt.git` it might fix the issue.

**Comment by shinra-electric** [comment]
This is odd, because the [.gitmodules](https://github.com/MaddTheSane/Boxer/blob/maddsV2/.gitmodules) file has the url as  `https://github.com/MaddTheSane/Boxer-MT32Emu.git` which should be fine...

**Comment by shinra-electric** [comment]
Ah, if you go to [here](https://github.com/MaddTheSane/Boxer-MT32Emu/blob/master/.gitmodules) you can see the old path.

Edit: Although after manually editing and retrying, it still fails with the same error. The old git address must be somewhere else too.

**Comment by kyods** [comment]
Go to Boxer/.git/modules/Vendor/Boxer-MT32Emu, and in the config file change:

```
[submodule "munt"]
	active = true
	url = git://github.com/munt/munt.git
```

to:

```
[submodule "munt"]
	active = true
	url = https://github.com/munt/munt.git
```

This should do the trick.

**Comment by shinra-electric** [comment]
Yes, that worked. Though I should mention that the modules folder is only created after trying to update the submodules once.

**Comment by MaddTheSane** [comment]
This should be fixed by commit 5f209fdd299f3a315464b6f82c3757b0dc20953c.
